wiki/redirect.yaml

```diff
@@ -497,102 +503,102 @@
-"guides/compressing_files":                                   "Guides/Compressing_Files"
-"guides/cropping_with_complex_backgrounds":                   "Guides/Cropping_with_Complex_Backgrounds"
-"guides/cropping_with_simple_backgrounds":                    "Guides/Cropping_with_Simple_Backgrounds"
-"guides/custom_hit_sound_library":                            "Guides/Custom_Hit_Sound_Library"
-"guides/custom_sample_overrides":                             "Guides/Custom_Sample_Overrides"
-"guides/getting_songs_from_video_games":                      "Guides/Getting_Songs_From_Video_Games"
-"guides/getting_your_map_modded":                             "Guides/Getting_Your_Map_Modded"
-"guides/getting_my_map_modded":                               "Guides/Getting_Your_Map_Modded"
-"guides/gl_support_issues":                                   "Guides/GL_Support_Issues"
-"guides/how_to_get_your_map_ranked":                          "Guides/How_To_Get_Your_Map_Ranked"
-"guides/how_to_get_my_map_ranked":                            "Guides/How_To_Get_Your_Map_Ranked"
-"guides/how_to_play_osu!mania":                               "Guides/How_to_Play_osu!mania"
-"guides/how_to_run_osu!_on_your_mac_without_using_boot_camp": "Guides/How_to_Run_osu!_on_Your_Mac_Without_Using_Boot_Camp"
-"guides/how_to_time_songs":                                   "Guides/How_to_Time_Songs"
-"guides/how_to_use_the_offset_wizard":                        "Guides/How_to_Use_the_Offset_Wizard"
-"guides/live_streaming_osu!":                                 "Guides/Live_Streaming_osu!"
-"guides/live_streaming":                                      "Guides/Live_Streaming_osu!"
-"guides/making_properly_centered_spinners":                   "Guides/Making_Properly_Centered_Spinners"
-"guides/make_properly_centered_spinners":                     "Guides/Making_Properly_Centered_Spinners"
-"guides/music_theory":                                        "Guides/Music_Theory"
-"guides/osu!mania_mapping_guide":                             "Guides/osu!mania_Mapping_Guide"
-"guides/recording_osu!":                                      "Guides/Recording_osu!"
-"guides/searching_and_downloading_beatmaps":                  "Guides/Searching_and_Downloading_Beatmaps"
-"guides/setting_the_offset_on_the_correct_beat":              "Guides/Setting_the_Offset_on_the_Correct_Beat"
-"guides/starting_a_beatmap_project":                          "Guides/Starting_a_Beatmap_Project"
-"guides/tablet_purchase":                                     "Guides/Tablet_Purchase"
-"guides/tips_and_tricks_on_skill_improvement":                "Guides/Tips_and_Tricks_on_Skill_Improvement"
-"guides/videos_from_youTube":                                 "Guides/Videos_From_YouTube"
 "adding_custom_hit_sounds":                                   "Guides/Adding_Custom_Hit_Sounds"
+"guides/adding_custom_hit_sounds":                            "Guides/Adding_Custom_Hit_Sounds"
```

Comment:
are these redirects actually needed where the only difference is case?

Reply:
Yes, since if someone were to type `guides/Adding_Custom_Hit_Sounds` (with a lowercase G), `guides/adding_custom_hit_sounds` (with all lowercase), or any in letter case (except for the ones used by the folder) it would 404.

I had thought of a possible fix for this that would remove most of these, but it may cause some unintended things to happen (mainly, the subtitle would be using the wrong case since it appears to be using the path). That being said, I was going to ask about using lowercase letters for all folder names (so osu!web could strtolower the path).

Reply:
Tracking at https://github.com/ppy/osu-web/issues/1817